static/index.html

```diff
@@ -8,6 +8,7 @@
     <link href="styles/rewards/claim-table.css" rel="stylesheet" />
     <link href="styles/rewards/media-queries.css" rel="stylesheet" />
     <link href="styles/rewards/light-mode.css" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Instead of importing the whole font, you should export the single svg and paste it into the html instead. Make sure to wrap it inside of another div and then you have more flexibility to style it. 

It's a lot more efficient compared to downloading all of the icons and then using a single one.